src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1135,6 +1136,21 @@ public string FileName
             }
         }
 
+        /// <summary>
+        ///     A unique id representing a workspace that was created from an in-memory graph content.
+        ///     This is usefull if you need to check if the current workspace was initially created from
+        ///     a specific graph content.
```

Comment:
can you update this comment to explain what the difference between this ID and the other ID property are? Why would I use one or the other?

Reply:
Yes will do and actually I think I'll make this internal ; it's too specific to our use case.

Reply:
I was going to ask if `internal` is a better option. Also since the DynamoCore binary is already internally visible so we should still be able to access it in unit tests